strings-csharp/ReplaceSpecialCharactersInString/ReplaceSpecialCharactersInString/StringReplacerBenchmarks.cs

```diff
@@ -0,0 +1,77 @@
+﻿using BenchmarkDotNet.Attributes;
+
+namespace ReplaceSpecialCharactersInString
+{
+    public class StringReplacerBenchmarks
+    {
+        private string originalString = "";
+
+        [Params(' ')]
+        public char charToReplace;
```

Comment:
1. charToReplace and charToReplace can be a normal class property like originalString, Params attribute is not required.
2. We should run each replace method atleast 50-100 times, for that we can create a Params call iteration and use a for loop inside each method to call the method for n times.